### DISCUSSION TITLE
If you have Mac OS, please run all tests.

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#53] [General]
I my Mac OS intel test with zipkin doesn't work, but functionality does, so need someone who has Mac to check this.

**Comment by drakon660** [comment]
@razvangoga @Mrxx99 do you have mac os?

**Reply by razvangoga** [reply]
I have an M1 MBP - will run the tests this week - don't have the proj setup there yet and i'm a bit caught up with work until friday)

**Reply by razvangoga** [reply]
nice avatar btw :))

**Reply by drakon660** [reply]
It is not professional as yours, but Donatello was my favourite ninja turtle in early 90's when saw them.

**Reply by razvangoga** [reply]
i was a Rafaello fan 🗡️

**Reply by drakon660** [reply]
tough guy

**Reply by Mrxx99** [reply]
> @razvangoga @Mrxx99 do you have mac os?

No
